### PR TITLE
Make sure ceph-mds is deployed (it's needed for manila)

### PR DESCRIPTION
**Pull request by jsuchome**
As we need only one ceph-mds node, let's take it first and we can
both make sure that there's one deployed and skip those complicated
checks for conflicts using the extra method.

i know it's embarassing, 3rd PR to do the same task ...

### CODE REVIEW

**Comment by jsuchome** [comment]
[#cloud8 Make OpenStack barclamps work for SP3](https://trello.com/c/DxMDobCg/33-cloud8-make-openstack-barclamps-work-for-sp3)

**Review by tserong — DISMISSED** [review]
Simpler is good :-)

**Review by Itxaka — DISMISSED** [review]

**Comment by jsuchome** [comment]
Gating is failing because of https://bugzilla.suse.com/show_bug.cgi?id=1046008 , problems with ceph backend in cinder

File: crowbar_framework/app/models/ceph_service.rb [hunk 1]
```diff
@@ -108,25 +108,6 @@ def proposal_dependencies(role)
 
   # Helper to find a suitable node for ceph-mds role.
   # ceph-mds conflicts with ceph-osd, and should not be assigned to the cluster node
```
Comment (dirkmueller) [inline crowbar_framework/app/models/ceph_service.rb]: can you remove those comments alongside of the function removal?

**Review by dirkmueller — CHANGES_REQUESTED** [review]

File: crowbar_framework/app/models/ceph_service.rb (hunk 1 above)
Comment (jsuchome) [reply crowbar_framework/app/models/ceph_service.rb]: done

**Comment by jsuchome** [comment]
@dirkmueller your comment has been addressed...

**Comment by jsuchome** [comment]
@Itxaka @tserong needs a new review after the force push...

**Comment by Itxaka** [comment]
@jsuchome arent you on PTO? Go away! :P

**Review by Itxaka — APPROVED** [review]

**Review by s-t-e-v-e-n-k — APPROVED** [review]

**Review by skazi0 — APPROVED** [review]
lgtm

**Review by tserong — APPROVED** [review]

**Review by dirkmueller — APPROVED** [review]

**Pull request merged**